packages/react-scripts/package.json

```diff
@@ -107,7 +107,8 @@
     "i18next": "^15.1.3",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
-    "react-i18next": "^10.11.0"
+    "react-i18next": "^10.11.0",
+    "react-router": "^5.0.1"
```

Comment:
I wonder if this is necessary since it's not a peer dep?

Reply:
i got a dep not found error with my linked cra app before i added the dep.
if there's something else that works, or it's a fluke, i could remove it, but that was my experience